### PR TITLE
:tada: Add delay interactions trigger

### PR DESCRIPTION
**Pull request by hirunatan**
> Please provide enough information so that others can review your pull request:

> Explain the **details** for making this change. What existing problem does the pull request solve?

> Screenshots/GIFs

### CODE REVIEW

File: frontend/src/app/main/ui/viewer/shapes.cljs [hunk 1]
```diff
@@ -173,6 +187,11 @@
           svg-element? (and (= :svg-raw (:type shape))
                             (not= :svg (get-in shape [:content :tag])))]
 
+      (mf/use-effect
+        (fn []
+          (let [sems (on-load shape)]
+            #(doall (map tm/dispose! sems)))))
```
Comment (niwinz) [inline frontend/src/app/main/ui/viewer/shapes.cljs]: in this case is better tu use `run!`

Comment (niwinz) [reply frontend/src/app/main/ui/viewer/shapes.cljs]: without doall

Comment (hirunatan) [reply frontend/src/app/main/ui/viewer/shapes.cljs]: done

**Comment by hirunatan** [comment]
duplicated

**Pull request closed without merging**